cmd/uplink/cmd/metaget.go

```diff
@@ -0,0 +1,85 @@
+// Copyright (C) 2019 Storj Labs, Inc.
```

Comment:
I would prefer to name this file `meta_get.go`, I have the feeling that it's easy to find out its scope (i.e. `meta`).

Reply:
Would that be consistent with the rest of the code? I see a lot of files following the `alllowercase` convention and only a few using `all_lower_case`.

Reply:
No idea what's the most consistent way, I gave my feedback on what I easily read.